### DISCUSSION TITLE
Keep sending Message until an acknowledgement is not received back.

### DISCUSSION THREAD

**Discussion by BforAzure2** [nextgenhealthcare/connect#4805] [Q&A]
Hello Mirth Gurus, 

We have a scenario where we want to ensure an acknowledgement must be received on an MLLP connection destination, and if not received no new message should be sent or keep trying sending old message.  Only HL7 acknowledgement message should be sent back all other response messages will be treated as not sent. 

How we can achieve this in Mirth? As in TCP Sender settings we can't see any such config?

**Comment by tonygermano** [comment]
That would be the "Validate Response" setting in the Destination Settings. Description from the User Guide and tool tip:

>Select Yes to validate the response. Responses can only be validated if the response
transformer's inbound properties contains a Response Validation section. If validation fails,
the message will be marked as queued or errored. For additional information, see Data
Types.

**Reply by BforAzure2** [reply]
@tonygermano Thank you. but I didn't get it.  How can I add response validation section. Can you elaborate please.

**Reply by tonygermano** [reply]
Mirth does the validation for you. There's nothing to do for validation except leave that setting turned on. Everything else in your question is handled by the destination queue settings.
